framework/Models/EnsembleModel.py

```diff
@@ -103,38 +104,41 @@ def localInputAndChecks(self,xmlNode):
         # get model name
         modelName = child.text.strip()
         # create space of the allowed entries
+        self.modelsInputDictionary[modelName] = {'TargetEvaluation':None,'Instance':None,'Input':[],'Output':[],'metadataToTransfer':[]}
```

Comment:
I would add a comment here explaining that you're setting up the structure of `modelsDictionary` to mirror `modelsInputDictionary`, just for ease of reading for the next user. Not required, just a thought in case you have to make other changes.

Reply:
Done